GETTING_STARTED_WITH_CELESTIA.md

```diff
@@ -0,0 +1,293 @@
+ ✔ Container integrations-celestia-validator-1  Started                                                                                                                                                                                  0.1s
+ ✔ Container integrations-celestia-node-0-1     Started                                                                                                                                                                                  0.2s
+waiting for container 'celestia-node-0' to become operational...
+[2025-07-31 12:05:14] health == 'starting': Waiting for celestia-node-0 to be up and running...
+[2025-07-31 12:05:17] health == 'starting': Waiting for celestia-node-0 to be up and running...
+[2025-07-31 12:05:20] celestia-node-0 is healthy
+ ✔ Celestia devnet containers are ready.
+```
+
+### Running the Rollup
+
+Clean the database to avoid any issues with rollup from MockDa, if it has been run previously
+
+```bash,test-ci,bashtestmd:exit-code=0
+$ make clean-db 
+```
+
+Now run your rollup with the `celestia_da` feature enabled:
+
+```bash,test-ci,bashtestmd:long-running,bashtestmd:wait-until=rest_address
+$ cargo run --no-default-features --features=celestia_da,mock_zkvm -- --rollup-config-path=configs/celestia/rollup.toml --genesis-path=configs/celestia/genesis.json
+```
+
+The log output should indicate a healthy running rollup. Verify that the REST API is responding:
+
+```bash,test-ci,bashtestmd:compare-output
+$ curl -s http://127.0.0.1:12346/modules/value-setter/state/value
+{"value":null}
+```
+
+## Stage 2: Celestia Testnet
+
+### Stopping the Devnet
+
+First, stop the local devnet and clean the database if you previously ran on devnet:
+
+```bash
+$ make stop-celestia
+$ make clean-db
+```
+
+### Setting Up a Celestia Light Node
+
+For this tutorial, we'll use the [Mocha testnet](https://docs.celestia.org/how-to-guides/mocha-testnet). 
+You'll need to run a Celestia light node to connect your rollup to the network.
+
+**Note**: For production deployments, it's recommended to connect your light node to a reliable RPC provider or use a bridge node for enhanced reliability and performance.
+
+#### Installation and Setup
+
+Follow the Celestia documentation for detailed instructions:
+- [Install celestia-node](https://docs.celestia.org/how-to-guides/celestia-node)
+- [Setting up a Celestia light node](https://docs.celestia.org/how-to-guides/light-node)
+
+#### Optimizing Initial Sync
+
+To speed up the initial synchronization, you can configure your light node to start from a recent block:
+
+1. Visit the block explorer: https://mocha.celenium.io/
+2. Select a recent block and note its hash and height
+3. Update your light node configuration:
+   - `Header.TrustedHash`: Use the block hash from step 2
+   - `DASer.SampleFrom`: Use the block height from step 2
+
+**Important**: This optimization means you won't be able to start your rollup from blocks prior to the selected height.
```

Comment:
And so lets tell them to set the rollup start height in rollup.toml (or that constants file - im not sure which) to be set to something after this DA height.